app/ViewStores.tsx

```diff
@@ -40,19 +47,35 @@ export function ViewStores(props: {
       </div>
     );
   });
-  return (
-    <Wrapper>
-      {displayedStores}
-      <LargeButton
-        onClick={() => {
-          onCreateClick();
-        }}
-        backgroundColor={greyInactiveButton}
-      >
-        Create
-      </LargeButton>
-    </Wrapper>
-  );
+  if (storeIsSelected) {
+    return (
+      <Wrapper>
+        <UpdateStore
+          onBackClick={() => {
+            setStoreIsSelected(false);
+          }}
+          onUpdateStoreSuccess={() => {
+            setStoreIsSelected(false);
+            console.log("function to overwrite existing store data goes here");
+          }}
+          storeDetails={storeDetails}
+        ></UpdateStore>
+      </Wrapper>
+    );
+  } else
```

Comment:
this else block is redundant since you returned in the previous if block

Reply:
https://github.com/auberginearms/groceryDatabaseApp/pull/20/commits/eb4c72fb8bbbc523c7c5372750adde1655234793